gateway/src/main/java/org/georchestra/gateway/events/EventsListener.java

```diff
@@ -0,0 +1,31 @@
+package org.georchestra.gateway.events;
+
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONObject;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.core.MessageListener;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+@Slf4j(topic = "org.georchestra.gateway.events")
+public class EventsListener implements MessageListener {
+
+    private static Set<String> receivedMessageUid = new HashSet<String>();
+
+    private static Set<String> synReceivedMessageUid = Collections.synchronizedSet(receivedMessageUid);
```

Comment:
do we really need to deduplicate messages here? what are the chances that rabbit sends duplicate events

Reply:
Currently, for no reasons , event are sent from console to gateway more than one time